extensions/wikia/WikiFactory/Loader/WikiFactoryLoader.php

```diff
@@ -8,35 +7,6 @@
  * @author Krzysztof Krzyżaniak (eloy) <eloy@wikia-inc.com> for Wikia Inc.
  * @todo change use of mIsWikiaActive to a series of isClosed, isDeleted, etc. methods
  */
-
-ini_set( "include_path", "{$IP}:{$IP}/includes:{$IP}/languages:{$IP}/lib/vendor:.:" );
-ini_set( "cgi.fix_pathinfo", 1);
```

Comment:
Not needed any more?

Reply:
https://github.com/Wikia/config/pull/2444/commits/87e63beb1fcab98deb170e6bed759fa69cabec3f - I've mentioned it here.

There is no need to set cgi.fix_pathinfo explicitly, as the secure behaviour is now the default. http://php.net/manual/en/ini.core.php#ini.cgi.fix-pathinfo